src/initializers/koa/error-handler.ts

```diff
@@ -9,6 +9,8 @@ export const isBlacklisted = (err: { status: number } = {} as any, config) =>
   // tslint:disable-next-line:triple-equals
   err.status && config.blacklistedErrorCodes.some(b => err.status == b);
 
+const onlyWarn = (err: { onlyWarn: boolean }) => !!err?.onlyWarn
```

Comment:
```suggestion
const logLevel = (err: { logLevel: LogLevels }) => !!err?.logLevel
```

Reply:
@nikostoulas Can you take a look at the latest implementation?

I was checking https://www.npmjs.com/package/http-errors documentation
![image](https://user-images.githubusercontent.com/15652566/182604070-1a95d945-5fe4-4f7e-85cd-8260d7e3d4ac.png)


So based on the `properties` field, in order to make this work with `ctx.assert` maybe only thing that's needed is to use it like this on the postings server side?
`ctx.assert(token, 401, { message: calculateMessage(), logLevel: 'WARN' });`

Reply:
Nice interface @Dimitris-Ilias .
Left a comment for the implementation details